src/pages/InfoPage.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import TeamInfo from '../components/TeamInfo';
+
+const InfoPage = () => (
+  <section>
+    <TeamInfo />
+  </section>
+);
```

Comment:
- [ ] According to the project requirements  🙃  Let's make sure the Details page stick to the original design as much as possible 🎨 🖌️  
![image](https://user-images.githubusercontent.com/79058364/217826455-b06fb502-f6f4-4064-8c34-ed92221dc9f1.png)

- You have to change your item text color, and background color, ... and make sure to add the right missing icons.

Your Implementation | Original Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/221356455-be7fcdef-addf-4904-b5d0-86b745b97575.png) | ![image](https://user-images.githubusercontent.com/79058364/217828693-b49d57ec-1cb0-48d3-83e8-743823b60f19.png)